### DISCUSSION TITLE
Plate elements and complex shapes

### DISCUSSION THREAD

**Discussion by linokl** [JWock82/Pynite#131] [General]
I am trying to incorporate plate elements to StandardSolver. Is it possible to combine plate elements and create a plate girder for example? I can see there is individual plate elements you can model and put opening in them, as provided in one of the example files. Not sure how I can combine them through common nodes or edges. For example two plates sharing common edge below.
<img width="505" alt="Screenshot 2022-10-08 at 11 26 27" src="https://user-images.githubusercontent.com/20646443/194703608-b94fa597-60c8-44ea-8e7e-5d8f89514058.png">

Regards
Lino

**Comment by JWock82** [comment]
Hi Lino,

It is possible, and I've done it for shear walls with flanges and openings. One of these day I may get an example published when I find the time.

Here are the basic steps:


  *   Create a rectangular mesh for each component of the model (flange, web, flange).
  *   Use the same mesh size for each mesh to ensure plates and nodes line up.
  *   Use the `x_control` and `y_control` variables in the `RectangleMesh` objects to ensure that all the nodes and plates line up with each other where you need meshes to intersect. For example, web openings in a `RectangleMesh` would need to be matched with control points in the flange meshes.
  *   Add all meshes to the model and then merge duplicate nodes. Look at the circular bin with conical hopper example. It shows how to merge two meshes into one.

Hope this helps,

Craig Brinck

From: linokl ***@***.***>
Sent: Saturday, October 8, 2022 6:34 AM
To: JWock82/PyNite ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [JWock82/PyNite] Plate elements and complex shapes (Discussion #131)


I am trying to incorporate plate elements to StandardSolver. Is it possible to combine plate elements and create a plate girder for example? I can see there is individual plate elements you can model and put opening in them, as provided in one of the example files. Not sure how I can combine them through common nodes or edges.

Regards
Lino

-

**Reply by linokl** [reply]
Hi Carig,

Thanks for the reply. I will try to implement it.

Thank you
Lino

**Comment by linokl** [comment]
also an example of a simply supported plate girder (with a web and a flange) with a simple point load on it..or something similar will be great help.

Regards
Lino
